### PR TITLE
Completed dubstrike/twitter-dark-mode#53

### PR DESCRIPTION
**Pull request by DylanPlemmons**
Styled the homepage when not logged in.

### CODE REVIEW

**Comment by pugson** [comment]
Thanks!

File: dist/min/darkmode.css [hunk 1]
```diff
@@ -0,0 +1 @@
...1P7mM7E,html{background-color:#0e131a;color:#fff}._2vY0UT8z:active,._3bf6snf7,.qb_4uBPl{background-color:rgba(29,161,242,.15)}._1YVNMg8K ._29kyA7wX{background-color:#657786;border-color:#657786}._266FL8X1:before{border-color:#657786}._1y-7iMRs:active,._1y-7iMRs:focus,._1y-7iMRs:hover,.ktZMpANQ{background-color:rgba(101,119,134,.25)}._1n7-Ka2i{background:#0e131a;border-top:1px solid #657786}._1MWHskJm,._1NUqA_Ya,._38myriBx,._38myriBx:active,._38myriBx:focus,._38myriBx:hover{background-color:#1c9dec}._1u_kMV_N,._3WJqTbOE{color:#fff}._1d_6kzhv,._1s7qNPIc,._2jEwv4_q{box-shadow:0 1px 0 rgba(101,119,134,.25)}._1_vKaW5Q,._1gq1kIcW,._1Oxj60h6,._1qygzEvp,._1XHWlb5c,._2hx-WgQl,._2nUWv41O,._2OhV6WSs ._1eF_MiFx ._2sSKjr8W,._2SfRhtNW,._2Sszw7pC,._3-utd0lH,._3DRCrDG0,._3facEWb0,._3facEWb0:active,._3facEWb0:focus,._3facEWb0:hover,._16q9OV8F,._22uh2-tW,._33HzS5QJ,._qZ42ApI,.FaFY-n1u,.FsR6j-G7,.pQ0LYqFV,.qb_4uBPl{border-color:rgba(101,119,134,.25)}._1HXcreMa,._3ZSf8YGw{color:#1c9dec}._3fv5_7QL{border-bottom:1px solid #1b95e0}._1_KafmK5{border-bottom:1px solid rgba(101,119,134,.25)}._3Ko_vUO1,._15-p_S0N{border:1px solid rgba(101,119,134,.25);color:#999}._3QPrjhX2,.lIgYF7yz{border-color:transparent}}
```
Comment (pugson) [inline dist/min/darkmode.css:1]: @DylanPlemmons Is the `.gitignore` not working? This file should not be commited to the repo.

Comment (DylanPlemmons) [reply dist/min/darkmode.css:1]: idk, I upload through the github website (too dumb to understand git) and drag the scss file and dist folder in and commit it.

Comment (pugson) [reply dist/min/darkmode.css:1]: Oh, I see. [Here's an easy to understand git tutorial](https://www.codeschool.com/courses/try-git) if you wanna try it.

Comment (DylanPlemmons) [reply dist/min/darkmode.css:1]: Alright thanks!

**Pull request merged**